apps/web/components/ui/NavbarBottom.vue

```diff
@@ -1,14 +1,16 @@
 <template>
   <nav class="z-50 w-full fixed bottom-0 left-0 flex flex-row items-stretch md:hidden" data-testid="navbar-bottom">
     <SfButton
-      v-for="{ label, icon, path } in items"
+      v-for="{ label, icon, link } in items"
       :key="label"
       variant="tertiary"
       :class="[
-        'py-1 flex flex-col h-full w-full rounded-none bg-primary-700 text-white hover:text-white hover:bg-primary-800 active:text-white active:bg-primary-900',
-        { 'text-white bg-primary-900': $route.path === path },
+        '!p-1 !pt-3 flex flex-col h-full w-full rounded-none bg-primary-700 text-white hover:text-white hover:bg-primary-800 active:text-white active:bg-primary-900 !text-xs !font-base',
+        { 'text-white bg-primary-900': $route.path === link },
       ]"
-      @click="handleClick(path)"
+      size="sm"
+      :tag="NuxtLink"
+      :href="link"
```

Comment:
Suggestion:
I'd say it's always better to use `to` prop of NuxtLink instead of native href attr

```suggestion
      :to="link"
```

Reply:
any links that can backup this claim? I can't find any info that `to` is preferred attribute

Reply:
https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/76/commits/9e27cc5745b3519163e2bbf9acf70b1535f205eb

Reply:
Yeah, there was some security issues reported by sonarcloud when used with `href`.

Reply:
Ah, that's was it - so maybe it's not connected to NuxtLink by itself, but to our tooling 😄